### PR TITLE
with `emoji` icons

### LINKED ISSUE #54: No one want to read text on notifications. Let's add opt-in emoji icons
❌ Error
⚠️ Warn
✅ Success

options.emoji: true

### PR DESCRIPTION
**Pull request by a-x-**
closes #54

Great UX coming us. User can understand what is happened w/o focusing notification now.
Well known icons deal with our primitive and fast part of brain directly w/o extra distraction to reading

We can be confused when error caused, but we did not read the text sometimes

### CODE REVIEW

**Comment by a-x-** [comment]
Thank you much!
Can you release a version also? =)

**Comment by Turbo87** [comment]
released as v1.8.0 a few seconds ago

File: index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ WebpackNotifierPlugin.prototype.compileMessage = function(stats) {
 
     } else if (!this.lastBuildSucceeded || this.options.alwaysNotify) {
         this.lastBuildSucceeded = true;
-        return 'Build successful';
+        return (hasEmoji ? '✅ ' : '') + 'Build successful';
```
Comment (mAAdhaTTah) [inline index.js]: This uses `hasEmoji` before it's assigned, so it'll always be false.

**Comment by Gvozd** [comment]
@mAAdhaTTah fixed at 1.10.1

**Pull request merged**